src/test/java/net/kaczmarzyk/spring/data/jpa/Customer.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2014-2022 the original author or authors.
- *
+ * <p>
```

Comment:
why did you add it?

Reply:
generally the license headers are managed automatically via Maven license plugin